### PR TITLE
V1.19.x

### PR DESCRIPTION
**Pull request by srini100**
patch release

### CODE REVIEW

File: gRPC-C++.podspec [hunk 1]
```diff
@@ -23,15 +23,15 @@
 Pod::Spec.new do |s|
   s.name     = 'gRPC-C++'
   # TODO (mxyan): use version that match gRPC version when pod is stabilized
-  # version = '1.19.0'
+  # version = '1.19.1'
```
Comment (sanjaypujare) [inline gRPC-C++.podspec]: Nit: looks like we are already using grpc_version so this line and the previous comment line should be removed (and the generation script modified not to look for this)

File: package.xml [hunk 2]
```diff
@@ -13,8 +13,8 @@
  <date>2018-01-19</date>
  <time>16:06:07</time>
  <version>
-  <release>1.19.0</release>
-  <api>1.19.0</api>
+  <release>1.19.1</release>
+  <api>1.19.1</api>
```
Comment (sanjaypujare) [inline package.xml]: @stanley-cheung do you know why we are updating both the api and release versions? If the api has not changed, we should not bump it up, right? Not a big issue for this PR though...

File: gRPC-C++.podspec (hunk 1 above)
Comment (srini100) [reply gRPC-C++.podspec]: cc @muxi 
This can be resolved in master for next release.

File: package.xml (hunk 2 above)
Comment (stanley-cheung) [reply package.xml]: Yea I will make a note of these and fix it in `master`. But this `api` version doesn't really matter.

Comment (sanjaypujare) [reply package.xml]: Does it mean it can be removed?

Comment (stanley-cheung) [reply package.xml]: That I am not sure, there's a format for these PECL extension.

**Pull request merged**